kvstore/sql.go

```diff
@@ -189,11 +211,15 @@ func (o SQL) Add(key string, val string) error {
 		return err
 	}
 
-	// nolint:gosec
-	// o.TableName has been checked by isSafeTblName on init
-	q := fmt.Sprintf("INSERT INTO %s(key, vals) VALUES(?, ?)", o.TableName)
+	query := sq.Insert(o.TableName).Columns("kv_key", "kv_val").
+			Values(key, val).PlaceholderFormat(o.Placeholder)
 
-	_, err := o.DB.Exec(q, key, val)
+	query_text, args, err := query.ToSql()
+	if err != nil {
+		panic(err)
```

Comment:
Any reason to `panic()`?

Reply:
The error only currs if the query is malfomed this something that needs to be fixed entirely in this file.

Reply:
It makes sense if that's the only reason - apart from OOM, of course, which would kill the runtime anyway.